packages/xo-server-usage-report/src/index.js

```diff
@@ -476,24 +476,41 @@ class UsageReportPlugin {
 
   configure (configuration) {
     this._conf = configuration
+    const enabled = this.job !== undefined && this.job.enabled
```

Comment:
Maybe we could improve the plugins API to pass a boolean to `configure` indicating whether the plugin is loaded?

Reply:
If you want to add this feature, update `xo-server` and`xo-server-test-plugin` to reflect it.